### PR TITLE
Raw values in NW and Circle-Watchface

### PR DESCRIPTION
**Pull request by AdrianLxM**
Show raw in NW under "minutes ago" and on the watch (circle watchface).

3 raw values are shown. The leftmost is the oldest.
The values are from the last up to 15 minutes. It will show an "x" if the value is not present or older than 5 minutes on the rightmost, older than 10 minutes on the middle or older that 15 minutes on the leftmost slot.

Should it stay on the NW Home screen?

### CODE REVIEW

**Comment by AdrianLxM** [comment]
Thanks @francesc0-cgm for helping and testing and supplying the live NS test data!

**Comment by francesc0-cgm** [comment]
Thank you @adrianlxm for developing this

File: mobile/src/main/java/com/dexdrip/stephenblack/nightwatch/Rest.java [hunk 1]
```diff
@@ -35,6 +35,7 @@
         mUrl = prefs.getString("dex_collection_method", "https://{yoursite}.azurewebsites.net");
         PowerManager powerManager = (PowerManager) context.getSystemService(Context.POWER_SERVICE);
         this.wakeLock = powerManager.newWakeLock(PowerManager.PARTIAL_WAKE_LOCK, "rest wakelock");
+        wakeLock.acquire();
```
Comment (StephenBlackWasAlreadyTaken) [inline mobile/src/main/java/com/dexdrip/stephenblack/nightwatch/Rest.java]: hehehe wooops

**Comment by StephenBlackWasAlreadyTaken** [comment]
This looks great, thank you for putting all this time in. Everything looks alright from this end, let me know if you think it should get merged or if you were still working on it!

**Comment by francesc0-cgm** [comment]
Wakelocks added from @AdrianLxM gives stable updates about 2 mins later than NS when phone is sleeping.
Tested all the watchfaces: great job Adrian.

**Comment by AdrianLxM** [comment]
@StephenBlackWasAlreadyTaken, I'd just like to know if I can change the text in the white bar on round watchface with graph from 8sp to 10sp like it is on square watches. 
I have no round watch but on Photos from Jason Q. the text looks very small and looks like it would fit with 10sp on a lg g watch r.

**Comment by francesc0-cgm** [comment]
No problem at all on square watchface. Font is tiny but readable

**Comment by AdrianLxM** [comment]
@StephenBlackWasAlreadyTaken, works stable for Alessandro, Jason and me for about 6 days now. I'm not planning on adding anything more... If you want you can merge, Thank you.

**Comment by francesc0-cgm** [comment]
Please Stephen merge it :-)

**Comment by StephenBlackWasAlreadyTaken** [comment]
:-D Sorrrrrry!

**Pull request merged**